### PR TITLE
[workbench/fileActions] add focusActiveFileInExplorer action

### LINKED ISSUE #40924: Feature Request: Action to focus active editor file in explorer
I'm looking to setup a keybinding that will focus the current editor file in the explorer, giving me a visual frame for renaming/moving/adding this or nearby files. I currently accomplish this with a combination of `workbench.files.action.showActiveFileInExplorer` and `workbench.files.action.focusFilesExplorer`, but just focusing the files explorer scrolls to the top of the list.

### PR DESCRIPTION
**Pull request by spalger**
Fixes #40924

Adds a new action `workbench.files.action.focusActiveFileInExplorer` that functions identically to `workbench.files.action.showActiveFileInExplorer`, except it focuses the explorer after showing it, allowing use of the keyboard to navigate the file explorer from the context of the current file.

### CODE REVIEW

File: src/vs/workbench/parts/files/electron-browser/fileActions.ts [hunk 1]
```diff
@@ -1311,7 +1309,7 @@ export class ShowActiveFileInExplorer extends Action {
 	public run(): TPromise<any> {
 		const resource = toResource(this.editorService.getActiveEditorInput(), { supportSideBySide: true });
 		if (resource) {
-			this.commandService.executeCommand(REVEAL_IN_EXPLORER_COMMAND_ID, { resource });
+			this.commandService.executeCommand(REVEAL_IN_EXPLORER_COMMAND_ID, resource, this.focusSelected);
```
Comment (spalger) [inline src/vs/workbench/parts/files/electron-browser/fileActions.ts]: Note the change in arguments here. Passing `resource` as an property on an object isn't actually supported by the reveal in explorer command, seems to be a bug introduced by https://github.com/Microsoft/vscode/pull/40168.

**Comment by spalger** [comment]
Turns out the "show explorer" action does the trick!

**Comment by gillyb** [comment]
Can we re-open this (and possible merge) ?
I ran into this when looking for the same solution, and Cmd+Shift+E ("Show explorer") doesn't seem to do the trick for me. It just focuses on the explorer, but doesn't go to the currently open file.. :/

**Comment by spalger** [comment]
Reopened and merged master @gillyb, until https://github.com/Microsoft/vscode/pull/40876 is merged I can't use VSCode on a daily basis, so I didn't realize it didn't work.

**Comment by gillyb** [comment]
Thanks a lot @spalger !
I'm also trying to transition to VSCode, but there are some minor issues holding me back. So I'm going over the issues looking for what I need, and possibly "fixing" what isn't.. :)

**Comment by isidorn** [comment]
Thanks for the PR, however this functionality is already part of VSCode - simply trigger the action "Show explorer"
If the "Show explorer" action is not reveal the currently open file then please file a seperate issue with reproducable stpes.

Due to the reasons above closing this as will not merge.

**Pull request closed without merging**